e2e_test.go

```diff
@@ -157,8 +170,14 @@ func runIntegrationTest(t *testing.T, testFile string) {
 	}
 }
 
-func skipTest(testFile, testName string) bool {
+// skipTest returns whether we should skip a testfile
+// if disableGroupImport is set to true, then we don't skip that test
+func skipTest(testFile, testName string, disableGroupImport bool) bool {
 	fullName := filepath.Join(filepath.Base(testFile), testName)
```

Comment:
I would expected skipTest to take test file and test name as inputs, an parameter like disableGroupImports seems non-obvious. Would adding additional flags to gopatch also increase the number of arguments to this function?

One possible approach would be to split the cases factoring in disableGroupImports into separate table tests. What do you think?

Reply:
+1 to this. The `disableGroupImport` parameter effectively overriding the `testsToSkip` thing is gnarly.